src/akamai/lib/AkamaiHDPlayer.as

```diff
@@ -263,21 +263,17 @@ package
     {
       if (event.info.code == "NetStream.Buffer.Full")
       {
-        if (_initialPlay)
```

Comment:
Is removing this check ok? is NetStream.Buffer.Full only fired once when the video is initially loaded?